cmd/token/create.go

```diff
@@ -41,8 +40,6 @@ k0s token create --role worker --expiry 10m  //sets expiration time to 10 minute
 `,
 		PreRunE: checkCreateTokenRole,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// Disable logrus for token commands
-			logrus.SetLevel(logrus.WarnLevel)
```

Comment:
The token commands could also do `logrus.SetOutput(os.Stderr)`

Reply:
stderr/warn is the k0s default, so I think it would be redundant here?

Reply:
Then it shouldn't matter if it outputs warnings as long as the token goes to stdout and everything else to stderr.